kernel/fde/optee/fde_key_handler_ta_type.h

```diff
@@ -0,0 +1,95 @@
+/* SPDX-License-Identifier: BSD-2-Clause
+ *
+ * Copyright 2025, Canonical Ltd.
```

Comment:
nitpick: afaiu the policy we use now for copyright is that the year is not needed